### PR TITLE
[1LP][RFR]Fix conversion host rails command

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================

__Fixing__

1) Adding ConversionHost.delete_all rails command to avoid duplicate conversion host
2) Updating is_plan_started method for progress card widget and wait_for fail_condition to avoid failure during navigation to the Migration RequestDetails Page, was timing out before with TimedOutError
3) Reducing number of Test Cases collected against whole of V2V using ONE_PER_TYPE in place of ONE_PER_VERSION

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f6d0eb5b78b6ac56043d2ac11507654fa5f7cf3c:

`cfme/tests/v2v/test_v2v_migrations_ui.py`:
- :heavy_exclamation_mark: Line 483:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/v2v/test_v2v_migrations_single_vcenter.py [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+@pytest.mark.parametrize('power_state', ['RUNNING', 'STOPPED'])
+def test_single_vm_migration_power_state_tags_retirement(request, appliance, v2v_providers,
+                                    host_creds, conversion_tags,
+                                    form_data_vm_obj_single_datastore,
+                                    power_state):
+    """
+    Polarion:
+        assignee: kkulkarn
+        casecomponent: V2V
+        initialEstimate: None
```
Comment (jawatts) [inline cfme/tests/v2v/test_v2v_migrations_single_vcenter.py]: **Required** Initial estimate needed here

File: cfme/tests/v2v/test_v2v_migrations_single_vcenter.py [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+    src_vm = form_data_vm_obj_single_datastore.vm_list[0]
+    migrated_vm = get_migrated_vm_obj(src_vm, v2v_providers.rhv_provider)
+    assert src_vm.mac_address == migrated_vm.mac_address
+
+
+def test_migration_long_name(request, appliance, v2v_providers, host_creds, conversion_tags):
+    """Test to check VM name with 64 character should work
+
+    Polarion:
+        assignee: None
```
Comment (jawatts) [inline cfme/tests/v2v/test_v2v_migrations_single_vcenter.py]: Need assignee here as well as casecomponent

Comment (mshriver) [reply cfme/tests/v2v/test_v2v_migrations_single_vcenter.py]: casecomponent isn't required yet, I'm updating that in separate PR with bulk setting of all the currently missing ones.

**Pull request merged**